### DISCUSSION TITLE
psql binary missing.

### DISCUSSION THREAD

**Discussion by alyxdeburca** [akiver/cs-demo-manager#770] [Q&A]
I have psql Version 16.2 installed and added to PATH. CSDM still says "PSQL binary not found" though. I'm on MacOS Sonoma

**Comment by akiver** [comment]
what's the output of `psql --version` from a terminal?

**Reply by alyxdeburca** [reply]
psql (PostgreSQL) 16.2

**Reply by akiver** [reply]
what shell do you use?

**Reply by alyxdeburca** [reply]
zsh. I also added the folder to /etc/paths

**Reply by akiver** [reply]
could you upload your log file at `~/.csdm/csdm.log`?

**Reply by alyxdeburca** [reply]
No such file, Could it be in a different location on mac?

**Comment by alyxdeburca** [comment]
[csdm.log](https://github.com/akiver/cs-demo-manager/files/14388239/csdm.log)

**Reply by akiver** [reply]
For some reason, your `PATH` variable can't be found.

Does running `zsh -ilc env;` contains the `PATH` variable? If so does it contains the path where `psql` is installed?

If by any chance you have node.js installed on your machine, what's the output of the following command?

```shell
node -e "console.log(require('node:os').userInfo().shell)"
```

**Reply by akiver** [reply]
@alyxdeburca does it has been resolved?
